test/mocks/runtime/mocks.cc

```diff
@@ -14,7 +16,11 @@ MockRandomGenerator::MockRandomGenerator() { ON_CALL(*this, uuid()).WillByDefaul
 
 MockRandomGenerator::~MockRandomGenerator() {}
 
-MockSnapshot::MockSnapshot() { ON_CALL(*this, getInteger(_, _)).WillByDefault(ReturnArg<1>()); }
+MockSnapshot::MockSnapshot() {
+  ON_CALL(*this, getInteger(_, _)).WillByDefault(ReturnArg<1>());
+  ON_CALL(*this, featureEnabled(_, An<uint64_t>()))
+      .WillByDefault(Invoke(this, &MockSnapshot::inheritedFeatureEnableStringUint64));
```

Comment:
Nit: Can you rename this to `featureEnabledDefault`? I think that would be a bit clearer.

Reply:
Thanks! Renamed in the latest diff.